src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -223,7 +319,9 @@ public RenameDriver CreateFieldRenamer(RecordType parameters, DPath pathToRename
             ** but that we don't return any display names for them. Thus, we clone a PowerFxConfig but without 
             ** display name support and construct a resolver from that instead, which we use for the rewrite binding.
             */
-            return new RenameDriver(parameters, pathToRename, updatedName, this, CreateResolver(Config.WithoutDisplayNames()), CreateBinderGlue());
+
+            // $$$ Config.WithoutDisplayNames()
```

Comment:
>  // $$$ Config.WithoutDisplayNames() [](http://example.com/codeflow?start=10&length=37)

@lesaltzm  - I'm passing all tests with this removed ... so maybe it wasn't necessary, or we may have a test hole?

If we do need this, I can solve it by passing a flag through CreateResolver (rather than creating whole new resolver)

Reply:
Yeah, looks like a test gap: I took another look at it, and added #562, removes the need for this.